### PR TITLE
Ilan's edits

### PR DESCRIPTION
**Pull request by petrelharp**
Ilan made a pass through the manuscript and handed it to me. I believe he was working on the most recent version, so there's no merging I could have messed up.

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -428,6 +377,70 @@ \subsection*{The Species Catalog}
...
+and \textit{Ns}, remain unchanged \citep[see the \texttt{SLiM} manual and][]{uricchio2014robust}.
+Such down-scaled simulations are not completely equivalent to simulating 
+all individuals in the population, and may lead to subtle differences.
+However, since many sequence-based measures of population diversity remain unchanged when rescaling in this fashion,
+this practice is very common when using forward-time simulators, such as \texttt{SLiM}.
+
+We validated our implementation of the \texttt{SLiM} engine by comparing
+estimates of several population genetic summary statistics for neutral simulations generated by both \texttt{SLiM}
+and \texttt{msprime} under different down-scaling factors
+(Figures \ref{fig:slim_val_map} and \ref{fig:slim_val_nomap}). This comparison indicates that scaling down the simulated
```
Comment (grahamgower) [inline popSimManu.tex]: A lot more validation was done than indicated by these two plots (more summary statistics, and for more demographic models). Maybe the language could be adjusted to reflect that?

File: popSimManu.tex [hunk 2]
```diff
@@ -428,6 +377,70 @@ \subsection*{The Species Catalog}
...
+of $50$ ($Q=50$ in \texttt{SLiM}) produces patterns of both diversity and linkage disequilibrium that are
+qualitatively similar to those observed under the coalescent (i.e. \texttt{msprime}), when both are simulated under a recombination map
+(Figure \ref{fig:slim_val_map}).
+We do observe a slight departure in values of Tajima's D when scaling down by a factor of $50$,
+likely due to the subtle differences implied by down-scaling.
+Interestingly, this effect of down-scaling is more prominent and extends to other sequence-based statistcs
+when data sets are simulated under a uniform recombination rate (Figure \ref{fig:slim_val_nomap}).
+Importantly, both comparisons validate the equivalence of \texttt{SLiM} and \texttt{msprime} when no down-scaling is applied,
+and they suggest that down-scaling population sizes can significantly reduce running time at a fairly modest cost to accuracy. 
+However, since the effects of rescaling are not fully understood, then relatively small scale factors should be used.
```
Comment (grahamgower) [inline popSimManu.tex]: Relative to what? The time span of the simulation? Ne? If Ne, then which population?

File: popSimManu.tex [hunk 3]
```diff
@@ -1152,10 +1173,10 @@ \section*{Appendix: Calculating coalescence rates}
 should be easier to infer from genomic data than aspects of population demography
 that do not affect effective population size.
 An analogous observation holds for populations of changing size,
-if we define the "coalescence rate" of a given demographic model
+if we define the ``coalescence rate" of a given demographic model
```
Comment (grahamgower) [inline popSimManu.tex:1198]: Shouldn't this be ` ``coalescence rate''`, with two single quotes on the right?

File: popSimManu.tex [hunk 4]
```diff
@@ -1152,10 +1173,10 @@ \section*{Appendix: Calculating coalescence rates}
...
 at a particular point back in time
 to be the rate of coalescence of remaining lineages
-and define the "coalescence effective size" at that time,
+and define the ``coalescence effective size" at that time,
```
Comment (grahamgower) [inline popSimManu.tex:1201]: Same comment here about single quotes on the right.

File: response-to-reviewers.md [hunk 5]
```diff
@@ -20,10 +20,21 @@ We have also added a bit more to the discussion (below "next steps") on this poi
...
+However, we felt that performing adequate simulations with selection would require
+a substantial amount of additional work and would significantly expand the scope of
+the manuscript.
+Furthermore, discussing selection would distract from the current relatively simple focus
+on inference of demographic history.
+We definitely intend to include selection (and expand the available generic models) in future work,
+as we now discuss in greater detail in the Discussion ("next steps").
+
+Consequently, we decided to add an example for possible use of SLiM in neutral simulations (see "simulation engines" section in the results and Supplementary Figures S1 and S2).
+Our focus there was to show that under neutral simulations, SLiM produced data set that were consistent with the coalescent simulations of msprime.
```
Comment (grahamgower) [inline response-to-reviewers.md]: s/produced data set/produced **a** data set/

**Review by grahamgower — APPROVED** [review]
Looks good! I left a few minor comments for the next round of edits.

File: popSimManu.tex (hunk 1 above)
Comment (petrelharp) [reply popSimManu.tex]: Could you suggest the change? Or even do it and push to this pr?

File: popSimManu.tex (hunk 3 above)
Comment (petrelharp) [reply popSimManu.tex:1198]: This worjs also.

**Comment by petrelharp** [comment]
Want to make those edits you suggest, @grahamgower?

**Comment by grahamgower** [comment]
Sure. [Here's a diff.](https://github.com/popsim-consortium/manuscript/files/4553699/minor-edits.txt) (I don't have push access, so can't just append to this branch.)

**Comment by petrelharp** [comment]
> I don't have push access

Oh, I see. Um, seems like maybe you should, but I'll just merge this in for the moment.

File: popSimManu.tex (hunk 2 above)
Comment (gphocs-dev) [reply popSimManu.tex]: I agree that the language I used was vague. I like your edit here.

File: popSimManu.tex (hunk 1 above)
Comment (gphocs-dev) [reply popSimManu.tex]: Suggested edit:
We validated our implementation of the \texttt{SLiM} engine by comparing estimates of several population genetic summary statistics for neutral simulations generated by both \texttt{SLiM} and \texttt{msprime} under various demographic models and using different down-scaling factors. We show a summary of two such comparisons for the AncientEurasia\_9K19 model \citep{kamm2019efficiently} using the
HapMapII\_GRCh37 genetic map \citep{international2007second} and a uniform recombination rate (Figures \ref{fig:slim_val_map} and \ref{fig:slim_val_nomap}, resp.).
These comparisons indicate ...

**Comment by petrelharp** [comment]
I had to do some rebasing here. Looking good; merging.

**Pull request merged**